src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -774,16 +775,26 @@ class DocBaseViewer extends BaseViewer {
         // Only initialize the find bar if the user has download permissions on
         // the file. Users without download permissions shouldn't be able to
         // interact with the text layer
-        const canDownload = checkPermission(this.options.file, PERMISSION_DOWNLOAD);
-        if (!canDownload || this.getViewerOption('disableFindBar')) {
+        if (this.isFindDisabled()) {
             return;
         }
 
-        this.findBar = new DocFindBar(this.findBarEl, this.pdfFindController, this.pdfEventBus, canDownload);
+        this.findBar = new DocFindBar(this.findBarEl, this.pdfFindController, this.pdfEventBus);
         this.findBar.addListener(VIEWER_EVENT.metric, this.emitMetric);
         this.findBar.addListener('close', this.handleFindBarClose);
     }
 
+    /**
+     * Determines if findbar is disabled
+     *
+     * @private
+     * @return {boolean}
+     */
+    isFindDisabled() {
+        const canDownload = checkPermission(this.options.file, PERMISSION_DOWNLOAD);
```

Comment:
very minor: I think the intention with this `const` is to be more readable so I would probably add the negation in the const creation and name it as such.